src/tests/topp/FeatureFinderIdentification_1_output.featureXML

```diff
@@ -377,7 +377,7 @@
 			<UserParam type="float" name="leftWidth" value="2094.083251953125"/>
 			<UserParam type="float" name="rightWidth" value="2197.34326171875"/>
 			<UserParam type="float" name="peak_apices_sum" value="4.861266306152344e05"/>
-			<UserParam type="floatList" name="masserror_ppm" value="[456.256957954320967, -42.506989184944054]"/>
+			<UserParam type="floatList" name="masserror_ppm" value="[-1.0, -42.506989184944054]"/>
```

Comment:
Using -1 as magic constant could be a bit dangerous. Is this properly documented?

Reply:
Typically this score would not be used in downstream analysis tools but rather just for the users information. I will add documentation in the function.